src/controller/org.controller/org.middleware.js

```diff
@@ -14,6 +14,16 @@ function isOrgRole (val) {
   return true
 }
 
+function isUserRole (val) {
+  val.forEach(role => {
+    if (!CONSTANTS.USER_ROLES.includes(role)) {
+      throw new Error('User role \'' + role + '\' does not exist.')
```

Comment:
Issue #262 was pushed through recently to prevent some values from getting reflected back to the client. Can you confirm that this line doesn't introduce the same anti-pattern?